packages/commerce-sdk-react/src/auth/storage.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {BaseStorage, ServerStorage} from './storage'
+
+const {window} = global
+const key = 'key'
+const value = 'value'
+
+const testCases = [
+    {
+        description: 'ServerStorage works without options',
+        requiresWindow: false,
+        storageOptions: {},
+        validate: (storage: BaseStorage) => {
+            storage.set(key, value)
+            expect(storage.get(key)).toBe(value)
+            storage.delete(key)
+            expect(storage.get(key)).toBe('')
+        }
+    },
+    {
+        description: 'ServerStorage works with options',
+        requiresWindow: false,
+        storageOptions: {
+            keyPrefix: 'prefix',
+            keyPrefixSeperator: '$'
+        },
+        validate: (storage: BaseStorage) => {
+            storage.set(key, value)
+            expect(storage.get(key)).toBe(value)
+            // @ts-ignore
+            expect(Array.from(storage.map.entries())[0][0]).toBe(`prefix$${key}`)
```

Comment:
```suggestion
            expect([...storage.map.keys()]).toEqual([`prefix$${key}`])
```
Just a touch shorter. Also, subtly double checks that there's exactly one key!